app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  links = [
```

Comment:
This is not related to the changes here, but I've noticed that the links for all of the step-by-step screens are being defined in the view. This is something that should probably be moved into a presenter sometime.